README.md

```diff
@@ -33,10 +33,11 @@ updates:
   the content.
 1. When you're ready for a review, add a comment to the PR, remove any holds or
   "WIP" markers, and assign a reviewer/approver. See the
-  [Kubeflow contributor guide][kubeflow-contributor-guide].
+  [Kubeflow contributor guide](https://github.com/kubeflow/community/blob/master/CONTRIBUTING.md).
 
-If you need help with the GitHub workflow, take a look at the quick guide near
-the bottom of this page.
+If you need more help with the GitHub workflow, take a look at this
```

Comment:
Nit: slightly shorter version

If you need more help with the GitHub workflow, read this

Reply:
Done, except that I used "follow" instead of "read", because it's a guide that needs more active participation than just reading.